src/security/PeerConnector.cc

```diff
@@ -677,13 +677,13 @@ Security::PeerConnector::certDownloadingDone(SBuf &obj, int downloadStatus)
         if (!Ssl::findIssuerCertificate(cert, certsList, ctx)) {
             if (const auto issuerUri = Ssl::findIssuerUri(cert)) {
                 debugs(81, 5, "certificate " <<
-                       X509_NAME_oneline(X509_get_subject_name(cert), buffer, sizeof(buffer)) <<
+                       *cert <<
                        " points to its missing issuer certificate at " << issuerUri);
```

Comment:
line wrapping here is no longer necessary for at least the first two lines of this debugs(). similar for the other touched one below.

Reply:
Adjusted to avoid arguing. Commit 4c16b6c.